gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-all.zip
```

Comment:
If you teach gradle what the expected sha256sum of the distribution zip file should be it would be more secure.
You can run 
`./gradlew wrapper --gradle-version=8.1.1 --distribution-type=all --gradle-distribution-sha256-sum=5625a0ae20fe000d9225d000b36909c7a0e0e8dda61c19b12da769add847c975`
 to make sure the line  `distributionSha256Sum=5625a0ae20fe000d9225d000b36909c7a0e0e8dda61c19b12da769add847c975` 
is added to the `gradle-wrapper.properties` file.
The Checksums can be looked up at https://gradle.org/release-checksums/.
(The command would also potentially update the `gradle-wrapper.jar` file and `gradlew.bat` / `gradlew.sh` scripts to the latest version.)